Add review point guide

mijin85cho opened the pull request:
Signed-off-by: Mijin, Cho <mijin85.cho@samsung.com>

Add Review point guide to help reviewers.

@annie-samsung and @sagarkedari 
Please check the contents and leave comments if you want to add more history you remember.
This is not a guide to users, but to reviewers,
so you don't need to review this pages.
Just check if it is easy to refer when a reviewer wants to study how to review.

annie-samsung commented on reviewguide/review_points_guide.md:
```diff
@@ -0,0 +1,221 @@
+
+<a name="create"></a>
+
+## Created page
```
Can we say "Add new topic" "Add new page"

mijin85cho replied:
Done

annie-samsung commented on reviewguide/review_points_guide.md:
```diff
@@ -0,0 +1,221 @@
+
+   **Example :** Use "Check points" instead of "Checking points".
+   
+- After the heading, simple sentences to describe the basic knowledge are recommended.
```
Do we also mention to avoid articles as much as possible in Headings?

mijin85cho replied:
Updated the sentence. Would you please check?

annie-samsung commented on reviewguide/review_points_guide.md:
```diff
@@ -0,0 +1,221 @@
+   - [toc_all.md](#create_toc_all)
+   - [overview.md](#create_overview)
+- [Changed file name / delete the page](#change_delete)
+- [Code block](#codeblock)
```
Do we also mention about Code tag usage for API/Function names? What about Enum values?

annie-samsung replied:
Also do we mention guidelines for **Bold** usage? Ex. UI commands
What about Filenames?

mijin85cho replied:
I thought the style guide will work for it. I added the link to that part.

annie-samsung commented on reviewguide/review_points_guide.md:
```diff
@@ -0,0 +1,221 @@
+- [Images](#images)
+- [Related information](#related_info)
+   - [Dependencies](#related_info_dependencies)
+
```
How about mentioning the check for Note Tag?

mijin85cho replied:
I thought the style guide will work for it. I added the link to that part.

mijin85cho commented:
@annie-samsung I updated with your feedback and added more pages (about Web API and stg URL).
If you send me the checklist file, I'll compare it this this PR and update the md files for the missed things, as we said on chat.

mijin85cho commented:
The review point for guide is updated to add several points.

mijin85cho commented:
Hello, @asm-foysal 
I'm adding the guides for reviewers that they can check, and hope you can read the files also.
While reading the pages, if you're not sure with something, please leave your questions by comments.
I'll check them and enhance the pages with the questions.

asm-foysal reviewed (commented):
Noted, thank you so much for this guide Catherine.

mijin85cho commented:
If you have no more request to update this guide,
would you please merge this PR?

For the further review points you want to add,
I'll create a new PR at that time.

annie-samsung commented:
@asm-foysal do you have any comments for this PR, if not could you approve it, I shall merge it

asm-foysal reviewed (approved):
@annie-samsung I have no questions and am approving this PR for merging.
